app/assets/stylesheets/moj/_HOTFIX_KILLME.scss

```diff
@@ -16,9 +16,11 @@
   @include clearfix;
 }
 .grid-row {
-  @include container;
-  margin-top:$gutter-half;
-  margin-bottom:$gutter-half;
+  &:not(.grid-row-fix) {
```

Comment:
🗒  Reason for this change, is mainly todo with the fact that the design seem to be using CSS directly from the GOV CSS, whereas we seem to be patching, so in order to preserve the current behaviour for all the pages except the one I have edited, I've come up with this solution.